.github/workflows/build_hub.yml

```diff
@@ -60,9 +60,11 @@ jobs:
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
 
       - name: Build backend
+        if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
```

Comment:
It looks like we are doing more than just skip schema validation in this PR? Please update the PR description to include a description and reason for these changes or remove these changes from this PR?